packages/default-storage-backend/android/build.gradle

```diff
@@ -47,14 +47,19 @@ buildscript {
             ? rootProject.ext['kotlinVersion']
             : rootProject.hasProperty('AsyncStorage_kotlinVersion')
             ? rootProject.properties['AsyncStorage_kotlinVersion']
-            : '1.8.10'
+            : '1.9.20'
```

Comment:
Will this cause issues when `react-native` is still on 1.8? And what about older versions?

In `react-native-test-app`, we keep a list of compatible versions: https://github.com/microsoft/react-native-test-app/blob/e43b6a622bffaa76ce28e1ed1c4f036db6611342/android/dependencies.gradle#L6

Should we be doing the same here?

Reply:
Kotlin is being kept forward binary compatible, so It should be fine with older versions. Gradle will select latest kotlin version and evict the rest.

I do like your suggestion tho, I could add a map of supported KSP versions, so we can decide on it based on Kt version used

Reply:
The addition of the supported KSP versions map will be particularly valuable for legacy projects, providing a clear understanding of compatibility.